.travis/tag-filter.sh

```diff
@@ -0,0 +1,34 @@
+TAG_GIT_URL=$(cat $PACKAGE/meta.yaml | grep "git_url" | awk '{print $2}')
+TAG_GIT_REV=$(cat $PACKAGE/meta.yaml | grep "git_rev" | awk '{print $2}')
+TAG_PATTERN=${TAG_PATTERN:-"v[0-9]*\.[0-9]*"}
+TAG_TMP_DIR=$(mktemp -d)
+
+git clone -b "$TAG_GIT_REV" "$TAG_GIT_URL" $TAG_TMP_DIR && pushd $TAG_TMP_DIR
```

Comment:
Conda will download the git repository (in "bare" mode) to `conda/conda-bld/git_cache/github.com/SymbiFlow/yosys.git` -- I suggest we reuse this location so we don't end up downloading the git repository multiple times.

With a bare repository you set `GIT_DIR` to the correct location. IE
```
GIT_DIR=~/conda/conda-bld/git_cache/github.com/SymbiFlow/yosys.git git describe --long --tags
```
This also should be part of `conda-env.sh` script.

Reply:
I've modified the script to use `.../conda-bld/git_cache` directories but I'm not sure how should I make this part of `conda-env.sh`. I've added a `source .travis/tag-filter.sh` there but now it requires `PACKAGE` and `CONDA_PATH` to be set.